Assets/Scripts/Utility/MacroHelper.cs

```diff
@@ -367,31 +375,33 @@ private static string LocalReputation(IMacroContextProvider mcp)
             PlayerGPS gps = GameManager.Instance.PlayerGPS;
             PersistentFactionData factionData = Game.GameManager.Instance.PlayerEntity.FactionData;
             int rep = factionData.GetLegalReputation(gps.CurrentRegionIndex).value;
-            if (rep > 60)
-                return "Esteemed";
+            if (rep > 80)
+                return "revered";
+            else if (rep > 60)
+                return "esteemed";
             else if (rep > 40)
-                return "Honored";
+                return "honored";
             else if (rep > 20)
-                return "Admired";
+                return "admired";
             else if (rep > 10)
-                return "Respected";
+                return "respected";
             else if (rep > 0)
-                return "Dependable";
+                return "dependable";
             else if (rep == 0)
-                return "Common Citizen";
+                return "a common citizen";
             else if (rep < 0)
-                return "Undependable";
+                return "undependable";
             else if (rep < -10)
-                return "Scoundrel";
+                return "a scoundrel";
             else if (rep < -20)
-                return "Criminal";
+                return "a criminal";
             else if (rep < -40)
-                return "Villain";
+                return "a villain";
             else if (rep < -60)
-                return "Pond Scum";
+                return "pond Scum";
```

Comment:
Shouldn't `Scum` be lowercased as well? (`scum`)

Reply:
Well spotted.